app/middleware/csp.js

```diff
@@ -19,7 +19,7 @@ const CSP_SELF = ["'self'"]
 const frameAndChildSourceCardDetails = ["'self'", 'https://secure-test.worldpay.com',
   'https://centinelapi.cardinalcommerce.com']
 
-const imgSourceCardDetails = ["'self'", 'https://www.google-analytics.com', 'https://www.gstatic.com']
+const imgSourceCardDetails = ["'self'", 'https://www.google-analytics.com', 'https://www.gstatic.com', 'https://d106cm2l6ezls7.cloudfront.net']
```

Comment:
In case there are more 👀  on this: as discussed this will be useful for building it and proving the end to end but this domain will be provided through through the environment by TF before reaching production